strapi/strapi
# Date: Store & display ISO dates instead of datetimes

Signed-off-by: HichamELBSI <elabbassih@gmail.com>



### What does it do?

Send only the date for the date field type.

### How to test?

If you are using a production build or `yarn start` please make sure to rebuild the helper-plugin before:

```
cd /packages/core/helper-plugin
yarn build
```

### Related issue(s)/PR(s)

Fix #11509

Back end PR : https://github.com/strapi/strapi/pull/12234
